test/gtest/uct/test_p2p_err.cc

```diff
@@ -139,7 +147,7 @@ class uct_p2p_err_test : public uct_p2p_test {
         if (r->loopback) {
             /* In loop back IB TLs can generate QP flush error before remote
              * access error. */
-            ucs_log(UCS_LOG_LEVEL_ERROR, "Error on ep %p with status %s is handled",
+            ucs_log(UCS_LOG_LEVEL_DIAG, "Error on ep %p with status %s is handled",
```

Comment:
why we hide this error? this test does err and test it

Reply:
if this error is printed, we can't expect no errors in the test.

Reply:
but this test does err and expects to handle this

Reply:
in case the running test is local_access_error or remote_access_error, we don't expect any error. 
i can change it to expect only this error. wdyt?

Reply:
I think we have to print errors always for both local_access_error and remote_access_error since it's most likely app bag. how to handle it in test - actually does not matter, IMO we have to wrap and print such errors if test checks bad scenario.